JSONStorage.swift

```diff
@@ -20,6 +20,11 @@ public enum JSONStorageType {
     }
 }
 
+public enum UseReadMemoryCache {
+    case no
```

Comment:
I dont like the idea of having enum here. I would separate this to simple useReadMemoryCache and saveTimeInterval variables.

Reply:
But `saveTimeInterval` makes no sense when `useReadMemoryCache` set to false. But I agree this enum is also poor solution

Reply:
So lets keep previous solution and make a comment about that this property is not used when useReadMemoryCache is false.

Reply:
Ok Bro! 😸